src/Service/Context/PageContext.php

```diff
@@ -238,6 +246,57 @@ public function getAll() {
   }
 
   /**
+   * Get meta tags.
+   *
+   * @return array
+   *   Get meta tags.
+   */
+  public function getMetatags() {
+    $metatags = [];
+    foreach ($this->metatags as $metatagName => $metatagContent) {
+      $metatag = [
+        '#type' => 'html_tag',
+        '#tag' => 'meta',
+        '#attributes' => [
+          'itemprop' => 'acquia_lift:' . $metatagName,
+          'content' => implode(',', $metatagContent),
+        ],
+      ];
+      $metatags[] = [$metatag, $metatagName];
+    }
+    return $metatags;
+  }
+
+    /**
+   * Get Lift credential meta tags.
+   *
+   * @return array
+   *   Get meta tags.
+   */
+  public function getLiftMetatags($credential_settings) {
+    $metatags = [];
+    $metatagAcct = [
+    '#type' => 'html_tag',
+    '#tag' => 'meta',
+      '#attributes' => [
+        'itemprop' => 'acquia_lift:account_id',
+        'content' => $credential_settings['account_name'],
+      ],
+    ];
+    $metatags[] = [$metatagAcct, 'account_id'];
+    $metatagSite = [
```

Comment:
The metatags' config arrays for $metatag, $metatagAcct, and $metatagSite are identical in array structure, it looks like we are able to extract the logic into a separate private function and reduce the duplicated code.